README.md

```diff
@@ -1195,7 +1195,7 @@ Inherits from [Common Plugin](#common-plugin).
 #### Instance file structure
 
 By default, instance file are stored directly in project, under so called main AEM module usually named *aem*.
-Ensure having directory *aem/.aem* ignored in VCS and excluded from indexing by IDE.
+Ensure having directory *aem/.instance* ignored in VCS and excluded from indexing by IDE.
```

Comment:
finally decided to have two directories

`aem/.instance`
`aem/.environment`

instead of 

`aem/.aem/instance`
`aem/.aem/environment`

for having better visibility (easier to find it when starting working with GAP)
(just 2 git ignores instead of 1